packages/modal/README.md

```diff
@@ -0,0 +1,158 @@
+# @zendeskgarden/container-modal [![npm version](https://img.shields.io/npm/v/@zendeskgarden/container-modal.svg?style=flat-square)](https://www.npmjs.com/package/@zendeskgarden/container-modal)
+
+This package includes containers relating to modal in the
+[Garden Design System](https://zendeskgarden.github.io/).
+
+## Installation
+
+```sh
+npm install @zendeskgarden/container-modal
```

Comment:
Should this be plural?

Reply:
I've kept all other packages as singular so I think this should stay as is